docs-v2/api-reference/connection/post.mdx

```diff
@@ -19,7 +19,6 @@ The required fields depend on the type of token you are trying to import:
 {
 	"connection_id": "<connection-id>",          // Always required
 	"provider_config_key": "<integration-id>",   // Always required
-	"type": "OAUTH2" | "OAUTH1" ,                // OAUTH2 or OAUTH1
```

Comment:
@bastienbeurier Import connection no longer needs to specify a type as we can infer that from the provided provider